src/autolabel/tasks/base.py

```diff
@@ -86,8 +86,15 @@ def parse_llm_response(
             llm_label = self.NULL_LABEL_TOKEN
             logger.error(f"Error parsing LLM response: {response.text}")
         else:
-            successfully_labeled = True
             llm_label = completion_text.strip()
+            if self.config.task_type() in [
+                TaskType.CLASSIFICATION,
+                TaskType.ENTITY_MATCHING,
+            ]:
+                logger.warning(f"LLM response is not in the labels list")
```

Comment:
shouldn't we only be logging this warning when the label is not in the labels list (checked in next line)?

Reply:
Oh yep, good catch! I'll change it now